Services/Repository.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq.Expressions;
+
+
+namespace TodoNET6.Services
+{
+    public interface IEntity<TIndex>
+    {
```

Comment:
Yes because all entities must have an ID but constraining our IDs to a specific type is too rigid as the data layer for the API must interact with other CPC code so some may have IDs that are ints, strings, guids, etc.